### PR TITLE
Add Travis tests for the GraalVL native-image plug-in

### PR DESCRIPTION
**Pull request by ScalaWilliam**
Also move out the sbt downloader into the same directory as the
downloader.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ cache:
   - $HOME/.sbt
   - $HOME/.m2
   - $HOME/.ivy2
+  - $HOME/graalvm-ce-1.0.0-rc7
```
Comment (muuki88) [inline .travis.yml]: rc8 has been released two days ago :smile:

File: .travis.yml [hunk 2]
```diff
@@ -35,7 +36,8 @@ before_install:
     fi
 
 before_script:
-- curl -L --create-dirs -o $HOME/.sbt/launchers/1.2.6/sbt-launch.jar http://dl.bintray.com/sbt/maven-releases/org/scala-sbt/sbt-launch/1.2.6/sbt-launch.jar
+- .travis/download-sbt
+- . .travis/download-graalvm
```
Comment (muuki88) [inline .travis.yml]: Why two dots?

**Comment by muuki88** [comment]
No need for squashing 😉 GitHub has a button for this 😎

File: .travis.yml (hunk 2 above)
Comment (muuki88) [reply .travis.yml]: Thanks for the explanation 🤗 too long bash abstinence 😁
So this is needed in order to make the export work, right?

**Comment by muuki88** [comment]
You can merge after CI passed 👍

File: .travis.yml (hunk 2 above)
Comment (ScalaWilliam) [reply .travis.yml]: Indeed. There could be other ways but this is the neatest.

```bash
$ bash <(echo export x=y)
$ echo $x

$ source <(echo export x=y)
$ echo $x
y
$
```

**Pull request merged**